src/Application/Actions/Hooks/StripeUpdate.php

```diff
@@ -175,9 +175,9 @@ public function handlePayoutPaid(Event $event): Response
                     return $this->respond(new ActionPayload(400));
                 }
             }
-
-            $this->logger->info(sprintf('Acknowledging paid donations complete, persisted: %s', $count));
-            return $this->respondWithData($event->data->object);
         }
+        
+        $this->logger->info(sprintf('Acknowledging paid donations complete, persisted: %s', $count));
```

Comment:
We need to initialise `$count` unconditionally for it to be safe to use here.

Reply:
Good catch, thanks @NoelLH